packages/webapp/src/components/Task/TaskComplete/StepOne.jsx

```diff
@@ -88,6 +87,7 @@ export default function PureCompleteStepOne({
           farm,
           task: selectedTask,
           disabled: !changesRequired,
+          isModified: changesRequired,
```

Comment:
@craigyu 

It seems weird to send the same attribute down twice. But I don't immediately see a better option.
```
disabled: !changesRequired,
isModified: changesRequired,

// In PureIrrigationTask the following is always true when changesRequired is set. 
disabled={disabled || isModified} 
```

Reply:
unfortunately no, I didn't really want to add that either, but couldn't think of another way of doing it :/